packages/core/app-layout/src/components/aboutSection/AppAboutSection.vue

```diff
@@ -0,0 +1,241 @@
+          >{{ createdLabel }}: {{ created }}</span>
+          <span
+            v-if="created && displayModified"
+            class="about-section-timestamps-arrow"
+          >-></span>
+          <span
+            v-if="displayModified"
+            class="about-section-timestamps-modified"
+            data-testid="about-section-timestamps-modified"
+          >{{ modifiedLabel }}: {{ modified }}</span>
+        </div>
+
+        <div
+          v-if="$slots.actions"
+          class="about-section-actions"
+          data-testid="about-section-actions"
+        >
+          <slot name="actions" />
+        </div>
+      </div>
+    </template>
+
+    <template #body>
+      <div
+        v-if="isLoading"
+        data-testid="about-section-loading-skeleton"
+      >
+        <KSkeletonBox
+          height="2"
+          width="100"
+        />
+        <KSkeletonBox
+          height="2"
+          width="100"
+        />
+      </div>
+
+      <div v-else>
+        <p
+          v-if="description"
+          class="about-section-description"
+          data-testid="about-section-description"
+        >
+          {{ description }}
+        </p>
+
+        <div
+          v-if="$slots.default"
+          class="about-section-content"
+          data-testid="about-section-content"
+        >
+          <slot name="default" />
+        </div>
+
+        <hr
+          v-if="$slots['divider-section']"
+          data-testid="about-divider-section-separator"
+        >
+
+        <div
+          v-if="$slots['divider-section']"
+          class="about-divider-section"
+          data-testid="about-divider-section"
+        >
+          <slot name="divider-section" />
+        </div>
+      </div>
+    </template>
+  </KCard>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+
+const props = defineProps({
+  title: {
+    type: String,
+    default: '',
+  },
+  description: {
+    type: String,
+    default: '',
+  },
+  created: {
+    type: String,
+    default: '',
+  },
+  createdLabel: {
+    type: String,
+    default: 'Created',
+  },
+  modified: {
+    type: String,
+    default: '',
+  },
+  modifiedLabel: {
+    type: String,
+    default: 'Modified',
+  },
+  isLoading: {
+    type: Boolean,
+    default: false,
+  },
+})
+
+const displayModified = computed(() => {
+  return !!props.modified && props.modified !== props.created
+})
+</script>
+
+<style lang="scss" scoped>
+.kong-ui-app-about-section {
+  .about-section-title {
+    color: $kui-color-text;
+    font-size: $kui-font-size-40;
+    font-weight: $kui-font-weight-bold;
+    letter-spacing: $kui-letter-spacing-minus-20;
+    line-height: $kui-line-height-30;
+  }
+
+  .about-section-header-end {
+    display: flex;
+
+    .about-section-timestamps {
+      align-self: center;
+      color: $kui-color-text-neutral;
+      font-size: $kui-font-size-20;
+      line-height: $kui-line-height-20;
+
+      @media (max-width: $kui-breakpoint-phablet) {
```

Comment:
question: why are we using max-width instead of min-width? (mobile first 😄 )

If you need `max-width` you need to subtract a pixel
```suggestion
      @media (max-width: ($kui-breakpoint-phablet - 1px)) {
```

Reply:
I copy pastad and always forget which direction we're supposed to go. I will fix.